### DISCUSSION TITLE
Дизайн ИС

### DISCUSSION THREAD

**Discussion by DaovaJo** [DaovaJo/Isys_engineering#3] [Show and tell]
Выкладывайте сюда дизайн интерфейса, мы так быстренько найдем а конечный выложим в саму репу как окончательный вариант

**Comment by morozenka09** [comment]
[Окно регистрации_1.zip](https://github.com/DaovaJo/Isys_engineering/files/9734991/_1.zip)

**Comment by morozenka09** [comment]
[Окно чата_1.zip](https://github.com/DaovaJo/Isys_engineering/files/9735356/_1.zip)

**Comment by morozenka09** [comment]
[Окно чата_2.zip](https://github.com/DaovaJo/Isys_engineering/files/9739520/_2.zip)

**Comment by morozenka09** [comment]
[Иконки чата.zip](https://github.com/DaovaJo/Isys_engineering/files/9803247/default.zip)
В архиве 2 файла формата PNG и ICO
